packages/victory-group/src/victory-group.js

```diff
@@ -88,17 +89,21 @@ const VictoryGroup = (initialProps) => {
     name
   ]);
 
+  const userProps = React.useMemo(() => UserProps.getSafeUserProps(initialProps), [initialProps]);
```

Comment:
I'm not sure whether this `useMemo` is doing any good here. Since `initialProps` isn't stored in a state or ref value, I believe `useProps` will be re-calculated on every render. There is a fairly good explanation here. https://github.com/FormidableLabs/victory/issues/2040

For now, I wouldn't stress too much about memoizing this value since it isn't a very expensive function call. Unless you can see a performance difference, let's just remove the `useMemo` for now.